rgit.rb

```diff
@@ -4,12 +4,39 @@
 require_relative './Repo.rb'
 require 'pathspec'
 require 'pty'
-require 'pathspec'
+require 'pathname'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.